apps/site/pages/docs/faststore-api.mdx

```diff
@@ -0,0 +1,86 @@
+---
+title: 'FastStore API'
+sidebar_label: 'FastStore API'
+---
+
+import { Callout, Step, Steps } from 'nextra-theme-docs'
+
+<header>
+
+# FastStore API
+
+</header>
+
+**FastStore API** is an interface between your ecommerce platform and your store's frontend. It uses **[GraphQL](https://graphql.org/)**, a query language for APIs and a runtime for fulfilling queries, in order to expose structured data from everyday ecommerce tasks to frontend components.
```

Comment:
I really liked the official graphql link here. Do you think we could do that for mentions of GraphQL Concepts throughout the docs, such as Query, Mutation, Fragment, Schema, or do you think it would be too overwhelming for readers?